packages/currency/src/native.ts

```diff
@@ -57,12 +57,6 @@ export const nativeCurrencies: Record<NativeCurrencyType, (NativeCurrency & { na
       name: 'Fantom',
       network: 'fantom',
     },
-    {
-      symbol: 'BNB',
-      decimals: 18,
-      name: 'BNB',
-      network: 'bsctest',
-    },
```

Comment:
I don't understand this change

Reply:
it's just cleaning because it's not used.